redis/commands/core.py

```diff
@@ -314,6 +314,11 @@ class ManagementCommands:
     Redis management commands
     """
 
+    def auth(self):
```

Comment:
Can we add a docstring indicating the behaviour. This will help users recognize we have completeness - but not use the function.

Reply:
Also - can we add a test to validate the exception is thrown?